tensortrade/actions/dynamic_orders.py

```diff
@@ -83,14 +83,14 @@ def trade_sizes(self, trade_sizes: Union[List[float], int]):
             (x + 1) / trade_sizes for x in range(trade_sizes)]
 
     def get_order(self, action: int, portfolio: 'Portfolio') -> Order:
-        if action == 0:
+        if action == 0 or not all(self.actions[action]):
```

Comment:
This line does not.

Reply:
Why are we allowing invalid actions to be passed into `self.actions`?

Reply:
this action scheme produces always produces action 1 which has no pair with it. I assumes this corresponds to holding. Which effectively is the same as action 0. So there might be a better approach to avoid producing invalid action. However, without this line, it will fail currently right below it for action 1 (regardless how many actions the action scheme has in total)

Reply:
@notadamking updated the PR